index.html

```diff
@@ -47,8 +47,13 @@
 	<!-- endbuild -->
 
 	<style id="MJX-CHTML-styles">/* placeholder for MathJax */</style>
+	<!--europeana feedback widget -->
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@europeana/feedback-widget@0.3.2-rc.2/dist/europeana-feedback-widget.css">
 </head>
 <body>
+	<!--europeana feedback widget-->
```

Comment:
we should better move this just before the invocation of the script before the end  body tag